lib/plotting_functions.py

```diff
@@ -741,7 +193,7 @@ def make_polar_plot(wks, case_nickname,
     absmaxdif = np.max(np.abs(dif))
     absmaxpct = np.max(np.abs(pct))
 
-    if 'colormap' in kwargs:
+    '''if 'colormap' in kwargs:
```

Comment:
Is there a reason to keep this entire section of commented-out code?  If not then I might recommend just removing it entirely.